pkg/operator/status.go

```diff
@@ -289,9 +299,175 @@ func (optr *Operator) syncUpgradeableStatus() error {
 		coStatus.Message = "One or more machine config pools are updating, please see `oc get mcp` for further details"
 	}
 
+	// don't overwrite status if updating or degraded
+	if !updating && !degraded {
+		skewStatus, status, err := optr.isKubeletSkewSupported(pools)
+		if err != nil {
+			glog.Errorf("Error checking version skew: %v, kubelet skew status: %v, status reason: %v, status message: %v", err, skewStatus, status.Reason, status.Message)
+			coStatus.Reason = status.Reason
+			coStatus.Message = status.Message
+			return optr.updateStatus(co, coStatus)
+		}
+		switch skewStatus {
+		case skewUnchecked:
+			coStatus.Reason = status.Reason
+			coStatus.Message = status.Message
+			return optr.updateStatus(co, coStatus)
+		case skewUnsupported:
+			coStatus.Reason = status.Reason
+			coStatus.Message = status.Message
+			mcoObjectRef := &corev1.ObjectReference{
+				Kind:      co.Kind,
+				Name:      co.Name,
+				Namespace: co.Namespace,
+				UID:       co.GetUID(),
+			}
+			glog.Infof("kubelet skew status: %v, status reason: %v", skewStatus, status.Reason)
+			optr.eventRecorder.Eventf(mcoObjectRef, corev1.EventTypeWarning, coStatus.Reason, coStatus.Message)
+			return optr.updateStatus(co, coStatus)
+		case skewPresent:
+			coStatus.Reason = status.Reason
+			coStatus.Message = status.Message
+			mcoObjectRef := &corev1.ObjectReference{
+				Kind:      co.Kind,
+				Name:      co.Name,
+				Namespace: co.Namespace,
+				UID:       co.GetUID(),
+			}
+			glog.Infof("kubelet skew status: %v, status reason: %v", skewStatus, status.Reason)
+			optr.eventRecorder.Eventf(mcoObjectRef, corev1.EventTypeWarning, coStatus.Reason, coStatus.Message)
```

Comment:
just talked to ryan, and @QiWang19 can you remove this event? Since this is a supported state that might last fora  while, we don't think that filling the MCO's logs with it every 5 mins for a long time is going to be helpful and will create v v large log files.

Reply:
Removed.

Reply:
thanks Qi!